webui/react/src/pages/F_ExpList/CompareParallelCoordinates.tsx

```diff
@@ -264,7 +291,7 @@ const CompareParallelCoordinates: React.FC<Props> = ({
     });
   }, [selectedExperiments, selectedMetric, fullHParams, metricData, selectedScale, trials, data]);
 
-  if (!isLoaded) {
+  if (!isLoaded || isFetching) {
```

Comment:
state isn't necessary here -- just use whether or not the loadable is loading

Reply:
thats so true. will fix it shortly

Reply:
fixed